### ISSUE TITLE
Mixed Content Loading Problem in Chrome

### ISSUE THREAD

**Issue by CMCDragonkai** [sinhrks/cesiumpy#66]
Running the example on an HTTPS Ipython Server leads to this problem in Chrome and probably other browsers.

```
Mixed Content: The page at 'https://...ipynb' was loaded over HTTPS, but requested an insecure stylesheet 'http://cesiumjs.org/Cesium/Build/Cesium/Widgets/widgets.css'. This request has been blocked; the content must be served over HTTPS.
```

Basically we should always be loading content over HTTPS as that means it works for both HTTP and HTTPS sites.

**Comment by CMCDragonkai** [comment]
A second problem is that the Cesium JS object isn't loaded on an already created notebook (when running the example in a new cell), instead it requires reloading the entire notebook from scratch.

**Comment by sinhrks** [comment]
Thx for the report. Will fix based on your suggestion.

**Comment by sinhrks** [comment]
Css fix is applied to the latest master. Can you describe a small example for 2nd issue?

**Comment by CMCDragonkai** [comment]
Thanks can you release a new version with this fix? As for the 2nd issue, it's just when you put cesiumpy into its own notebook cell in ipython, it doesn't load immediately on an already loaded notebook. I have to refresh the entire browser.

**Comment by CMCDragonkai** [comment]
Actually that second issue may not be an issue anymore. Will let you know later. It's just a bit weird, as that code involving cesiumpy won't be shown unless you execute the cell (and this happens when you save the ipython and then come back to it), makes it difficult to be used to show other people by sharing a static notebook. I think the interactive plots deal with this by making the plot static by saving a static image of it. Is it possible to do this with cesiumpy?

**Comment by sinhrks** [comment]
Thx for update. Released v0.3.3, see https://pypi.python.org/pypi/cesiumpy

There is an issue to save a static image via cesiumpy (#31).  I think it is possible, but not digged in yet.
